Add keyboard references-vnext (#2380)
IgniteUI/igniteui-docfx: en/components/grid/grid.md

</div>
<div class="divider--half"></div>

## Keyboard Navigation

> Keyboard navigation in the IgxGrids provides a rich variety of keyboard interactions for the user. It enhances the accessibility of the IgxGrid and allows to navigate through any type of elements inside (cell, row, column header, toolbar, footer, etc.).

Reviewer: ```suggestion
Grid's keyboard navigation provides a rich variety of keyboard interactions for the user. It enhances accessibility and allows intuitive navigation through any type of elements inside (cell, row, column header, toolbar, footer, etc.).
```

> See these topics for more information [Grid Keyboard Navigation](../grid/keyboard-navigation.md), [TreeGrid Keyboard Navigation](../treegrid/keyboard-navigation.md), [Hierarchical Grid Keyboard Navigation](../hierarchicalgrid/keyboard-navigation.md).

Reviewer: ```suggestion
Check out these resources for more information:
 - [Grid Keyboard Navigation](../grid/keyboard-navigation.md)
 - [TreeGrid Keyboard Navigation](../treegrid/keyboard-navigation.md)
 - [Hierarchical Grid Keyboard Navigation](../hierarchicalgrid/keyboard-navigation.md)
 - [Blog post](https://www.infragistics.com/community/blogs/b/engineering/posts/grid-keyboard-navigation-accessibility-) - Improving Usability, Accessibility and ARIA Compliance with Grid keyboard navigation
```
